src/HTMLImage.js

```diff
@@ -1,7 +1,9 @@
 import React, { PureComponent } from 'react';
-import { Image, View, Text } from 'react-native';
+import { Image, View, Text, Dimensions} from 'react-native';
 import PropTypes from 'prop-types';
 
+const { width: screenWidth, height: screenHeight } = Dimensions.get('window')
```

Comment:
This is problematic because it doesn't take into account the possibility of screen rotations.
The best would be to have a functional component and [`useWindowDimensions`](https://reactnative.dev/docs/usewindowdimensions) hook. But that will require a full refactoring. Given we have a lot of pending PRs and no testing in this library yet, we'll wait for tests implemented before such refactoring.

In the meantime, you can:

- either use [`Dimensions.addEventListener`](https://reactnative.dev/docs/dimensions#addeventlistener) to subscribe to screen rotations;
- or use `Dimensions.get('window')` from the render function instead of calling it statically. That wouldn't be ideal, but that would also be cheaper in terms of complexity and rendering performance.